be/src/service/service.h

```diff
@@ -0,0 +1,17 @@
+//
+// Created by choury on 22-5-7.
+//
+
+#ifndef STARROCKS_SERVICE_H
+#define STARROCKS_SERVICE_H
+
+#include "util/logging.h"
+
+namespace starrocks {
+extern bool k_starrocks_exit;
+} // namespace starrocks
+
+void main_cn();
```

Comment:
would better use full name here.

BTW, do you think which name is better for cn and ce?

Reply:
These two methods begin with `start` is better?